src/test/OptionButtons/SortByButton.test.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from 'chai';
+import { chromium, Browser, Page, Locator } from 'playwright-chromium';
+import { describe, it, before, after } from 'mocha';
+import { createServer, ViteDevServer } from 'vite';
+import { options } from '../../test-kit/OptionButtons/constants';
+import { SortByButtonDriver } from '../../test-kit/OptionButtons/sortByButton-driver';
+
+describe('SortByButton Component', function () {
+   this.timeout(5000);
+
+   let server: ViteDevServer;
+   let browser: Browser;
+   let page: Page;
```

Comment:
Since this is something that you will do a lot, create a test utility that will have 2 functions, one init and one destroys. no need to do this here.